Source/Scene/parseBatchTable.js

```diff
@@ -166,8 +183,10 @@ function partitionProperties(batchTable) {
 
     const property = batchTable[propertyId];
     if (Array.isArray(property)) {
+      jsonProperties = defined(jsonProperties) ? jsonProperties : {};
       jsonProperties[propertyId] = property;
     } else {
+      binaryProperties = defined(binaryProperties) ? binaryProperties : {};
```

Comment:
would this cause an error in `transcodeBinaryProperties`? It seems to assume that the binary properties is defined

Reply:
I reverted this to always allocate the dictionary even if it's empty. Since `parseBatchTable` checks for `parseAsPropertyAttributes` up front, the case of an empty batch table won't be an issue for point clouds